.gitignore

```diff
@@ -22,4 +22,4 @@ node_modules
 
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
+yarn-error.log*
```

Comment:
Can you add a new line here so there's no changes in the .gitignore file?

Reply:
I'm not even sure why that happened... but it should be fixed now!